com_redhat_kdump/ks/kdump.py

```diff
@@ -83,10 +87,18 @@ def setup(self, storage, ksdata, payload):
             # Ensure that the amount is an amount in MB
             if self.reserveMB[-1] != 'M':
                 self.reserveMB += 'M'
-            new_args.append(' crashkernel=%s' % self.reserveMB)
+            new_args.append('crashkernel=%s' % self.reserveMB)
+
+        if self.enablefadump and os.path.exists(FADUMP_CAPABLE_FILE):
+            new_args.append('fadump=on')
 
         bootloader_proxy.SetExtraArguments(new_args)
 
+        # If the local storage object is not available, skip.
+        # FIXME: This is a temporary workaround.
```

Comment:
Hi, about this FIXME, if local storage object is not available (Anaconda uses the DBus Storage module), then the below code, which remove duplicated boot_args and append proper boot_args,  do we need to implement that again for the DBus Storage module?

Reply:
No, it is enough to set up the extra arguments above. The storage module will collect all arguments right before the bootloader installation, so it is safe to change them at this point. I have tested both versions and they worked fine.

Reply:
Thanks, but why say it's a temporary workaround then?

Reply:
Once Anaconda uses the DBus Storage module, we can completely remove the code for the local storage object, so it's only a temporary solution. I will open another PR later. We plan to release a new Anaconda tomorrow, could you please do a new release with this patch?

Reply:
Thanks, looks good to me, merged and will make a release now.

Reply:
Thank you!